### DISCUSSION TITLE
Microinteracciones

### DISCUSSION THREAD

**Discussion by gitbraga** [Telefonica/mistica-design#619] [Proposals]
Hola, pongo aqui un ejemplo que hice para empezar la discusión de tener microinteracciones en los elementos tapeables.

Hice un estudio durante el facelift, pero fue apenas para row list y algo como un snap card.

https://user-images.githubusercontent.com/62388286/179836377-60b8cfcc-c8e4-4fce-9aa2-dc1ebcccb09b.mov

**Comment by yceballost** [comment]
Genial, es algo que queríamos meter pronto. En nativo si que existen esos estado de press para los elementos como listas pero en web no. 

Tendriamos que unificar estos comportamientos en todos los componentes que tengan interacción y en todas las plataformas

**Comment by yceballost** [comment]
Vamos a hacer esta tarea aquí → https://github.com/Telefonica/mistica-design/issues/677

Cierro esta discussion
